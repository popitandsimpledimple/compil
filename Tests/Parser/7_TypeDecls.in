program main;
type 
    int = integer;
    MyString = string;
    MyDouble = double;
begin end.