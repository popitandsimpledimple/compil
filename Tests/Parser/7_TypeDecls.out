program
   main
   decls
      decls types
         decl type
            int
            type
               integer
         decl type
            mystring
            type
               string
         decl type
            mydouble
            type
               double
   block
      compounds statements
